app/assets/javascripts/search.coffee

```diff
@@ -42,3 +42,17 @@ $(document).ready ->
 # Handle presence of Javascript by turning off or on visibility of JS sensitive objects
 $('.js-disabled-object').css 'visibility': 'hidden'
 $('.js-enabled-object').css 'visibility': 'visible'
+
+# Change focus of cursor when search link is clicked on Manage Collections, Manage Variables, or Manage Services pages.
+$ ->
+ $('#search-focus').on 'click', ->
+    document.getElementById('keyword').focus()
```

Comment:
I can use either; Although, I haven't see jQuery in the app. I've only seen coffee. I may need you to point me in the right direction.